sc3/createenv.sh

```diff
@@ -64,13 +64,13 @@ git clone https://github.com/h5py/h5py.git && \
     cd .. && sudo rm -rf h5py
 
 # Setup virtualenv
+echo '' >> $HOME/.bashrc
 echo "source /usr/bin/virtualenvwrapper.sh" >> $HOME/.bashrc
 source $HOME/.bashrc
 
 mkvirtualenv --system-site-packages seismic
 
 echo "Installing passive seismic software....."
 workon seismic && \
-    git clone https://github.com/GeoscienceAustralia/passive-seismic && \
```

Comment:
Was this dropped unintentionally..?

Reply:
I see why. Looks great!

Reply:
Thanks @rh-downunder for the review. Merging.